modules/conversantAnalyticsAdapter.js

```diff
@@ -75,6 +75,7 @@ let conversantAnalytics = Object.assign(
   adapter({URL, ANALYTICS_TYPE}),
   {
     track({eventType, args}) {
+      utils.logMessage("CNVR Analytics: "+ eventType, Object.assign({}, args));
```

Comment:
Why is Object.assign needed?

Reply:
No clue, and also some extraneous log messages here to cleanup